homeassistant/components/mqtt/mixins.py

```diff
@@ -69,6 +75,9 @@
 
 MQTT_AVAILABILITY_LIST_SCHEMA = vol.Schema(
     {
+        vol.Optional(
+            CONF_AVAILABILITY_MODE, default=DEFAULT_AVAILABILITY_MODE
```

Comment:
given the use (and the fact that DEFAULT_AVAILABILITY_MODE isn't used anywhere else, why not declare `default= AVAILABILITY_LATEST` straight ahead ?